homework2/utilities/deleteMockedData.ts

```diff
@@ -0,0 +1,23 @@
+import { User, Group } from '../models/index';
+
+const hardDeleteUser = async (userID: string): Promise<number> => {
+    const res = await User.destroy({
+        where: {
+            id: userID
+        }
+    });
+    const affectedCount = res;
+    return affectedCount;
+};
+
+const deleteGroup = async (groupID: string): Promise<number> => {
+    const res = await Group.destroy({
+        where: {
+            id: groupID
+        }
+    });
+    const affectedCount = res;
+    return affectedCount;
+};
+
+export { hardDeleteUser, deleteGroup };
```

Comment:
all good and matches all requirements perfectly

one thing to look at is to mock the services rather than writing into the database

something like *taking from another repo

jest.mock('../CustomMiddleware/AuthMiddleware', () => ({
    Auth: jest.fn((req, res, next)=>next())
}));

jest.mock('../DataAccess/User', () => ({
    UserDAL: jest.fn().mockImplementation(() => ({
      Add: jest.fn((val)=>{
         return true;
      }),
      FindLike: jest.fn(()=>{
          return ["user1", "user2"]
      })
    }))
  }));